src/utils.py

```diff
@@ -365,36 +310,6 @@ def coco_evaluation(gt_filepath, prediction_filepath, image_ids, category_ids, s
     return cocoEval.stats[0], cocoEval.stats[3]
 
 
-def label(mask):
```

Comment:
@apyskir at the moment label_multiclass_image() uses this function.

Reply:
@taraspiotr Yes, but it was a duplicate